chef/site-cookbooks/wca/templates/wca.screenrc.erb

```diff
@@ -7,7 +7,7 @@ screen -t run
 <% if @rails_env == "development" %>
 stuff "RACK_ENV=development DATABASE_URL=<%= @db_url %> bundle exec rails server\012"
 <% else %>
-stuff "../scripts/deploy.sh rebuild_rails\012"
+stuff "../scripts/deploy.sh rebuild_rails update_docs\012"
```

Comment:
I believe this is the place responsible for the first deployment on a newly spun up server.

Reply:
I think so too.
The Regulations use their own recipe to be built, but it should be fine to get the docs here!